### PR TITLE
EntityReferenceAutocomplete: Autocomplete attribute added

### PR DESCRIPTION
**Pull request by vishalchandna1**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/418

## Screenshot / UI changes
screen shot 2018-08-27 at 4 42 27 pm

### CODE REVIEW

File: src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js [hunk 1]
```diff
@@ -272,14 +272,16 @@ class EntityReferenceAutocomplete extends React.Component {
     );
   };
 
-  renderInput = ({ InputProps, ref, label, ...other }) => (
+  renderInput = ({ InputProps, ref, label, autoComplete, ...other }) => (
```
Comment (dawehner) [inline src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Do you think there is ever a usecase where we want the browser based autosuggestion? I would argue that we don't have to make this configurable. 🤔

Comment (vishalchandna1) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Hi @dawehner 
I was also not so sure about it, so I added a generic case having `autoComplete` default value as `off`

Should we always keep `autocomplete` as `off`?

Comment (dawehner) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Yeah, I think the autocompletion by the browser is always annoying 👍

File: src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ import FormControl from '@material-ui/core/FormControl';
 import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
 import SchemaPropType from '../../05_pages/NodeForm/SchemaPropType';
 
-import api from './../../../utils/api/api';
+import api from '../../../utils/api/api';
```
Comment (mattgrill) [inline src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: Is there a reason we need to fix this here?

Comment (vishalchandna1) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: This is just to keep a consistent code like the other `import` statements.

File: src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js (hunk 1 above)
Comment (lauriii) [reply src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js]: I think we should force keep it always off 👍

**Pull request merged**